servlet/src/main/scala/org/http4s/servlet/BlockingHttp4sServlet.scala

```diff
@@ -106,13 +107,13 @@ object BlockingHttp4sServlet {
       ) {}
 
     @nowarn("cat=deprecation")
-    def build(implicit F: Sync[F]): BlockingHttp4sServlet[F] =
-      new BlockingHttp4sServlet(
-        httpApp,
-        BlockingServletIo(chunkSize.getOrElse(DefaultChunkSize)),
-        DefaultServiceErrorHandler,
-        dispatcher,
-      )
+    def build(implicit F: Async[F]): BlockingHttp4sServlet[F] =
```

Comment:
I wonder whether this can be made to compile as `Sync` still.  It's inherently synchronous.

Reply:
Up in the next PR, thank you